edx/edx-enterprise
# [ENT-618] Put view tests into separate files. (tech debt)

**Description:** This PR puts test classes that were in `tests/test_views.py` into separate files, depending on which class they were specifically testing.

**JIRA:** [ENT-618](https://openedx.atlassian.net/browse/ENT-618)

**Dependencies:** Currently based on the huge amount of tests that were added as part of ENT-486.

**Merge deadline:** Whenever.

**Testing instructions:**

1. This only puts tests into separate files to make it easier to manage, modify, add, and delete view tests. So just make sure the travis build is passing.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**
